# Image metadata consists of the following parameters
string edgeAPI
string customerName
string facility
string hostname
string space
string timestamp
int32[2] location
float32[2] absolutePosition
float32[3] orientation
float64 latitude
float64 longitude
string make
string model
float32 focalLength
string localPath
